lib/Sema/TypeCheckType.cpp

```diff
@@ -6146,6 +6146,28 @@ class ExistentialTypeSyntaxChecker : public ASTWalker {
     }
 
     if (auto *proto = dyn_cast<ProtocolDecl>(decl)) {
+        if (proto->existentialRequiresAny()) {
```

Comment:
Speaking of the error in question, where is the closing brace for this `if`?

Reply:
how did I miss that

Reply:
ready for another test run